src/js/stores/transforms/AppFormModelPostProcess.js

```diff
@@ -25,7 +29,7 @@ const AppFormModelPostProcess = {
       }
     }
 
-    var isEmpty = (Util.isArray(container.volumes) &&
+    let isEmpty = (Util.isArray(container.volumes) &&
```

Comment:
Jein, in a technical term yes, in a meaningful no, because:
```
if (container == null) {
  return;
}
```
can exit before the variabe isEmpty could be initialized.
The ```let``` doesn't hoist, so this should be the better selection here.